importer/yarn_parser.gd

```diff
@@ -8,7 +8,7 @@ static func read_line_with_indent(file: File):
 	var dedent = line.dedent()
 	return {
 		"indent": line.length() - dedent.length(),
-		"content": line.dedent().strip_edges()
```

Comment:
This call to `strip_edges` was necessary to avoid an issue with extra space on the end of the header end marker `---`, and keeps the lines fairly consistent wrt spacing, so I recommend adding it back :P

Reply:
Not sure how that got removed... so this was an oversight on my part.